src/main/kotlin/nl/avisi/kotlinwebtest/soap/Validation.kt

```diff
@@ -110,14 +110,14 @@ class SoapTestStep(testCase: TestCase) : TestStep<SoapStepRequest, SoapStepRespo
             endpoint ?: configuration.defaults.endpoint
 }
 
-class XPathValidator(val xpath: String, var value: Expression? = null) : Validator<SoapStepRequest, SoapStepResponse> {
+class XPathValidator(val xpath: String, val value: Expression) : Validator<SoapStepRequest, SoapStepResponse> {
```

Comment:
We might need to revert this to be able to write a better DSL (which unfortunately currently requires mutable state at many points). I'm willing to see how this turns out though.